skglm/experimental/tests/test_quantile_huber.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from numpy.testing import assert_allclose
+from sklearn.linear_model import QuantileRegressor
+from sklearn.datasets import make_regression
+from skglm.experimental.quantile_huber import SmoothQuantileRegressor
+
+
+@pytest.mark.parametrize('quantile', [0.3, 0.5, 0.7])
+def test_quantile_huber_matches_sklearn(quantile):
+    """Test that SmoothQuantileRegressor with small delta matches sklearn's
+    QuantileRegressor."""
+    X, y = make_regression(n_samples=1000, n_features=10, noise=0.1, random_state=42)
+
+    sk_est = QuantileRegressor(quantile=quantile, alpha=0.1, solver='highs').fit(X, y)
+    smooth_est = SmoothQuantileRegressor(
+        quantile=quantile,
+        alpha=0.1,
```

Comment:
@floriankozikowski alpha is chosen by hand, just check that it's not too high (= it does not give you coefficients all equal to 0)